WireAnalytics/Sources/WireAnalytics/AnalyticsManager.swift

```diff
@@ -84,14 +84,22 @@ public struct AnalyticsManager: AnalyticsManagerProtocol {
 
     private func updateUserProfile(_ userProfile: AnalyticsUserProfile) {
         analyticsService.changeDeviceID(userProfile.analyticsIdentifier)
-        analyticsService.setUserValue(userProfile.teamInfo?.id, forKey: "team_team_id")
-        analyticsService.setUserValue(userProfile.teamInfo?.role, forKey: "team_user_type")
-        analyticsService.setUserValue(userProfile.teamInfo.map { String($0.size.logRound()) }, forKey: "team_team_size")
+
+        for key in AnalyticsUserKey.allCases {
+            switch key {
+            case .teamTeamID:
+                analyticsService.setUserValue(userProfile.teamInfo?.id, forKey: key.rawValue)
+            case .teamUserType:
+                analyticsService.setUserValue(userProfile.teamInfo?.role, forKey: key.rawValue)
+            case .teamTeamSize:
+                analyticsService.setUserValue(userProfile.teamInfo.map { String($0.size.logRound()) }, forKey: key.rawValue)
+            }
+        }
```

Comment:
I find this a little confusing, I suggest to be simple:

```swift
analyticsService.setUserValue(userProfile.teamInfo?.id, forKey: AnalyticsUserKey.teamID.rawValue)
analyticsService.setUserValue(userProfile.teamInfo?.role, forKey: AnalyticsUserKey.teamRole.rawValue)
...
```

Reply:
Done: https://github.com/wireapp/wire-ios/commit/e4ab19dde01778aa71df214d8c89191ca7c343e6